Formula/openfpgaloader.rb

```diff
@@ -0,0 +1,26 @@
+class Openfpgaloader < Formula
+  desc "Universal utility for programming FPGA"
+  homepage "https://github.com/trabucayre/openFPGALoader"
+  url "https://github.com/trabucayre/openFPGALoader.git",
+    tag:      "v0.4.0",
+    revision: "1dbbd34420e00070bd0f58e0a91a653e12a87b97"
```

Comment:
Can we not use the source tarball instead?

Reply:
What's the preferred way? I've seen other recipes using GitHub tags, but also recipes using tarballs from GitHub.

Reply:
> What's the preferred way?

The tarball is preferred whenever possible, i.e., unless there are submodules to fetch which aren't included in GitHub source tarballs or the build system uses some `git` operations that we cannot avoid. Cloning is more expensive and slower, so we try to avoid it wherever possible.

Reply:
Thanks for the explanation, I'll provide an update.